cypress/support/bodyTestHelper.js

```diff
@@ -21,6 +21,70 @@ export const checkElementStyles = (elementString, text, color, fontFamily) => {
   shouldContainStyles(el, 'font-family', fontFamily);
 };
 
+export const getBlockData = (blockType, win) => {
+  let blockData;
+  const { blocks } = win.SIMORGH_DATA.data.content.model;
+
+  blocks.forEach(block => {
+    if (!blockData && block.type === blockType) {
+      blockData = block;
+    }
+  });
+  return blockData;
+};
+
+export const headlineDataWindow = () => {
```

Comment:
The name for these function could be clearer to explain what it's doing. 
E.g. `firstHeadlineBlockRendered`, `firstSubheadlineBlockRendered`, `firstParagraphBlockRendered`.